presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -111,6 +111,8 @@
 
     private final AtomicLong peakTaskUserMemory = new AtomicLong();
     private final AtomicLong peakTaskTotalMemory = new AtomicLong();
+    // TODO populate this value
```

Comment:
what's this?

Reply:
Didn't push an updated version of this. I populated it with the task statistics.